packages/node-cli/src/commands/file/import/ImportFilesAction.js

```diff
@@ -80,20 +86,28 @@ module.exports = class ImportFilesAction extends BaseAction {
 				.addParams(params)
 				.build();
 
+			let statusMessage = NodeTranslationService.getMessage(MESSAGES.IMPORTING_FILES);
+			if (this._calledFromUpdate) {
+				statusMessage = NodeTranslationService.getMessage(MESSAGES.UPDATING_FILE);
```

Comment:
This message is not the one showed in vscode while importing (updating or comparing) this is the message showed only in node-cli where we only have the file:import command.
I think we this is not necessary please remove it.